src/components/Header/LinksWrap/StyledLinkesWrap.ts

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import { backgroundColors, desktopSizes, weights, containerParams, neutralColors } from "../../../styledConstants";
+
+import { Link } from "react-router-dom";
+
+export const Wrapper = styled.div<{ $linkSections: string[] | undefined }>`
+position:absolute;
```

Comment:
if I understand correctly, it can be written shorter)
styled.div<{ $linkSections?: string[] }>`